Docs: document labels for trusted clusters (#27032)
gravitational/teleport: docs/pages/reference/cli.mdx

| Name | Default Value(s) | Allowed Value(s) | Description |
| - | - | - | - |
| <nobr>`--format`</nobr> | none | `text`, `json`, `yaml` | Output format |
| `--labels` | none | `text` | Sets token labels |

> | `--ttl` | 1h | relative duration like 5s, 2m, or 3h | Set expiration time for token |

Reviewer: ```suggestion
| `--ttl` | 1h | Duration like 5s, 2m, or 3h | Set expiration time for token |
```

Reviewer 2: I wouldn't say "set expiration time" - you're not setting an expiration time (that would be like saying "expire at 4pm tomorrow") - you're setting a validity period.

Author: FYI I didn't change it to say that, it's been the description for this flag as far back as the docs history goes. See L1816 where it was moved from. In any case, how about?

```suggestion
| `--ttl` | 1h | Duration like 5s, 2m, or 3h | Sets how long the token is valid for. |
```
